src/pages/LoginAndRegister/GuestAccountClaim.vue

```diff
@@ -9,6 +9,7 @@
 			</p>
 			<form id="guestAccountClaimForm" action="." @submit.prevent.stop="claimGuestAccount">
 				<kv-base-input name="firstName"
```

Comment:
FYI @emuvente These are the guest checkout component I found. If there's others in UI let me know, or could you add .fs-exclude classes to those inputs when you create them please